src/js/view/controls/controlbar.js

```diff
@@ -114,8 +114,8 @@ export default class Controlbar {
 
             const volumeButtonEl = volumeGroup.element();
             menus.push(volumeGroup);
-            // Use 'group' role so that voiceover identifies volume slider and button
-            setAttribute(volumeButtonEl, 'role', 'group');
+            // Use 'button' role so that voiceover identifies mute button as button
+            setAttribute(volumeButtonEl, 'role', 'button');
```

Comment:
I think this might bring back an issue in Safari where voiceover was not picking up and announcing the Volume Slider. This might have been because the slider is nested inside the button.

Reply:
Works fine locally :)